src/Compilers/CSharp/Portable/Binder/Binder_Patterns.cs

```diff
@@ -239,11 +245,23 @@ internal BoundExpression ConvertPatternExpression(TypeSymbol inputType, CSharpSy
             bool hasErrors,
             DiagnosticBag diagnostics)
         {
-            Debug.Assert(operandType != (object)null);
-
             var typeSyntax = node.Type;
+            var boundDeclType = BindPatternType(typeSyntax, operandType, ref hasErrors, out var isVar, diagnostics);
```

Comment:
>var [](start = 92, length = 3)

Please spell out the type.

Reply:
The use of `var` is recorded in [https://github.com/dotnet/roslyn/issues/23222](https://github.com/dotnet/roslyn/issues/23222) and addressed in [https://github.com/dotnet/roslyn/pull/23357/commits/553de58a82bc8f576c12918969a5aecaf6b9c309](https://github.com/dotnet/roslyn/pull/23357/commits/553de58a82bc8f576c12918969a5aecaf6b9c309) of [https://github.com/dotnet/roslyn/pull/23357](https://github.com/dotnet/roslyn/pull/23357).

---
In reply to: [156512281](https://github.com/dotnet/roslyn/pull/23209#discussion_r156512281) [](ancestors = 156512281)